bin/packages/build.js

```diff
@@ -123,10 +123,14 @@ function buildFileFor( file, silent, environment ) {
 	const buildDir = BUILD_DIR[ environment ];
 	const destPath = getBuildPath( file, buildDir );
 	const babelOptions = getBabelConfig( environment );
+	babelOptions.sourceMaps = true;
```

Comment:
Why set this here instead of defining it in our preset or in `getBabelConfig`?

Reply:
Yes, good catch. We should move there.

Reply:
I did that because it felt to me that the babel config should only be about the ESNext features and all the plugins needed and not about which file to output etc...

Reply:
`getBabelConfig` should be only updated, preset is more than okey :)

Reply:
After looking again, I'm convinced we should keep it here because the line after that `sourceFileName` is specific per file which confirms my original thinking

> I did that because it felt to me that the babel config should only be about the ESNext features and all the plugins needed and not about which file to output etc... And it needs too be shared between both babel configs we have.

Reply:
👍

Reply:
Do you remember why we override the default [sourceFileName](https://babeljs.io/docs/en/options#sourcefilename) value which absolute local path which is then being published to npm?

Reply:
I don't remember exactly. If we remove it and we can still navigate the code properly in dev mode in the browser's console, I'm fine with it.